### PR TITLE
ticdc: update troubleshoot for ddl skip

### PR DESCRIPTION
**Pull request by amyangfei**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions) **Add troubleshooting info for ddl skip**
- [x] v4.0 (TiDB 4.0 versions) **Add troubleshooting info for ddl skip**
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

ref: https://github.com/pingcap/ticdc/pull/1835

Add more instructions about how to find `start-ts` when cdc log doesn't contain it.

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/troubleshoot-ticdc.md [hunk 1]
```diff
@@ -311,9 +311,9 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
 ## 当 changefeed 的下游为类 MySQL 数据库时，TiCDC 执行了一个耗时较长的 DDL 语句，阻塞了所有其他 changefeed，应该怎样处理？
 
 1. 首先暂停执行耗时较长的 DDL 的 changefeed。此时可以观察到，这个 changefeed 暂停后，其他的 changefeed 不再阻塞了。
-2. 在 TiCDC log 中搜寻 `apply job` 字段，确认耗时较长的 DDL 的 `StartTs`。
+2. 在 TiCDC log 中搜寻 `apply job` 字段，确认耗时较长的 DDL 的 `start-ts`。如果在旧版本 TiCDC 中没有打印该 `start-ts`，可以查询 TiDB DDL history，找到该条 DDL 的 `binlog.TableInfo.update_timestamp` 字段，即为所需的 `start-ts`。
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:314]: "旧版本" 的 TiCDC 是什么版本呢？
```suggestion
2. 在 TiCDC log 中搜寻 `apply job` 字段，确认耗时较长的 DDL 的 `start-ts`。如果旧版本的 TiCDC 没有打印该 `start-ts`，可以查询 TiDB DDL history，找到该条 DDL 语句的 `binlog.TableInfo.update_timestamp` 字段，该字段即为所需的 `start-ts`。
```

File: ticdc/troubleshoot-ticdc.md [hunk 2]
```diff
@@ -311,9 +311,9 @@ TiCDC 对大事务（大小超过 5 GB）提供部分支持，根据场景不同
 ## 当 changefeed 的下游为类 MySQL 数据库时，TiCDC 执行了一个耗时较长的 DDL 语句，阻塞了所有其他 changefeed，应该怎样处理？
 
 1. 首先暂停执行耗时较长的 DDL 的 changefeed。此时可以观察到，这个 changefeed 暂停后，其他的 changefeed 不再阻塞了。
-2. 在 TiCDC log 中搜寻 `apply job` 字段，确认耗时较长的 DDL 的 `StartTs`。
+2. 在 TiCDC log 中搜寻 `apply job` 字段，确认耗时较长的 DDL 的 `start-ts`。如果旧版本的 TiCDC 没有打印该 `start-ts`，可以查询 TiDB DDL history，找到该条 DDL 语句的 `binlog.TableInfo.update_timestamp` 字段，该字段即为所需的 `start-ts`。
```
Comment (amyangfei) [inline ticdc/troubleshoot-ticdc.md:314]:
```suggestion
2. 在 TiCDC log 中搜寻 `apply job` 字段，确认耗时较长的 DDL 的 `start-ts`。如果旧版本（<=v4.0.13 或 <=v5.0.1）的 TiCDC 没有打印该 `start-ts`，可以查询 TiDB DDL history，找到该条 DDL 语句的 `binlog.TableInfo.update_timestamp` 字段，该字段即为所需的 `start-ts`。
```

Comment (amyangfei) [reply ticdc/troubleshoot-ticdc.md:314]: PTAL @TomShawn

Comment (TomShawn) [reply ticdc/troubleshoot-ticdc.md:314]: dev 版是最新开发版，应该不存在这个“旧版本”的历史问题。要不在 dev 中去掉“如果旧版本的 TiCDC 没有打印”这一句，在 release-x.0 文档中加上？

Comment (amyangfei) [reply ticdc/troubleshoot-ticdc.md:314]: 可以

Comment (amyangfei) [inline ticdc/troubleshoot-ticdc.md:314]:
```suggestion
2. 在 TiCDC log 中搜寻 `apply job` 字段，确认耗时较长的 DDL 的 `start-ts`。
```

**Comment by amyangfei** [comment]
ptal @TomShawn

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**